deps/wrapper.cc

```diff
@@ -79,6 +80,7 @@ void initialize_worker(
     options.raylet_ip_address = node_ip_address;
     options.metrics_agent_port = -1;
     options.startup_token = startup_token;
+    options.runtime_env_hash = runtime_env_hash;
```

Comment:
huh I actually came across this last week and tried harcoding `runtime_env_hash` but found it still didn't work

https://github.com/beacon-biosignals/ray/blob/4ceb62daaad05124713ff9d94ffbdad35ee19f86/src/ray/raylet/worker.h#L274-L277

Reply:
ah this actually corroborates what you saw in the logs about it not being properly propagated! `runtime_env_hash: 1462190447 cached runtime env hash: 0`

Reply:
ah you were on teh right track long before I was!